Services/CO.CDP.Organisation.WebApi/Model/ApprovableOrganisation.cs

```diff
@@ -0,0 +1,16 @@
+using CO.CDP.OrganisationInformation;
+
+namespace CO.CDP.Organisation.WebApi.Model;
+public record ApprovableOrganisation
```

Comment:
The endpoint we should be providing (GET /organisations) is more general purpose. I wouldn't couple this response name to "approvable" organisations only.

Reply:
I don't agree. The endpoint is specific for admin access and requires the extra fields